src/v0/destinations/clevertap/transform.js

```diff
@@ -378,8 +384,47 @@ const processEvent = (message, destination) => {
 const process = (event) => processEvent(event.message, event.destination);
 
 const processRouterDest = async (inputs, reqMetadata) => {
```

Comment:
can we remove async as no intermediate calls are happening

Reply:
Removed.